deploy/crds/bench_v1alpha1_bench_cr.yaml

```diff
@@ -21,3 +21,11 @@ spec:
     runtime: 57
     rw: write
     filesize: 1
+  sysbench:
+    tests:
```

Comment:
It'd be in best interest to add a boolean like enabled, which can be defaulted to false.

Reply:
The existing workloads don't have that. I was under the impression that if you didn't want to run the workload, you wouldn't have it in the CR in the first place. @jtaleric thoughts?

Reply:
The existing workloads do have a concept of # of pairs or clients. which I think might make sense here too...

```yaml
sysbench:
  clients: 0
  tests:
   ...
```

Thoughts? 

And yes, of course users could just have a workload defined in their CR. I would prefer they could go either route.

Reply:
@jtaleric Ack. I can add the enabled flag. However, I have a questions regarding number of client filed. Is that a hard requirement? I run my workload as a job. I don't think there is a way to set replicas with job, so the only way would be loop over with ansible and start multiple jobs, which I'm not sure is what we want. Thoguhts?